### ISSUE TITLE
Plot of AE history

### ISSUE THREAD

**Issue by Bio-data-tricks** [musikalkemist/generating-sound-with-neural-networks#3]
Hi Valerio, 
Thank you for your excelent work ! 
I would like plotting the evolution of loss and val_loss, how can I do this from your existing code ? 

`def train(x_train,y_train,x_test,y_test, learning_rate, batch_size, epochs):
    autoencoder = Autoencoder(
        input_shape=(888, 1),
        conv_filters=(32, 64, 64, 64),
        conv_kernels=(3, 3, 3, 3),
        conv_strides=(1, 2, 2, 1),
        latent_space_dim=2
    )
    autoencoder.summary()
    autoencoder.compile(learning_rate)
    autoencoder.train(x_train,y_train,x_test,y_test, batch_size, epochs)
    return autoencoder'

Thank you 

Nicolas

**Comment by Bio-data-tricks** [comment]
here is the solution !

# loss visualization
N = np.arange(0, EPOCHS)
plt.style.use("default")
plt.figure(figsize=(10, 7))
plt.plot(N, autoencoder.model.history.history["loss"], '--', label="train_loss")
plt.plot(N, autoencoder.model.history.history["val_loss"], label="test_loss")
plt.title("Loss history")
plt.xlabel("Epoch #")
plt.ylabel("Loss")
plt.legend(loc="upper right")
